packages/flame_jenny/jenny/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+
+## 1.0.0
+
+Initial release; this corresponds to YarnSpinner 2.0 with few extra features.
```

Comment:
This should be done together with the melos reelease, I'll copy this line and add it when I do the release. :)